deployments/helm/Chart.yaml

```
@@ -1,8 +1,8 @@
 apiVersion: v2
-appVersion: 1.16.0
+appVersion: v0.9
```

```suggestion
appVersion: v0.10.2
```
Or we can do this change when we update the stable release. WDYT?

---

When we will merge this PR, the latest helm chart publisher would run. Though it would create a new tar file on `charts` repo, it won't create a commit updating the versions in chart.yaml in kubearmor repo. So even if we keep it to v0.9 or unchanged here, the packaged chart will take version from STABLE_RELEASE which is v0.9.

Also, we won't ever need to manually update chart and app version. This would be taken care of in the stable release workflow.

---

Then if ain't ever be updated, shouldn't we set it to something else so as it is not misleading or suggests it's unamanaged?

---

Yeah I thought we could do it just this one time so that charts repo and the chart.yaml here seemed in sync. What else do you suggest :thinking: